dm/dm.go

```diff
@@ -166,9 +166,19 @@ func execute() {
 	switch args[0] {
 	case "templates":
 		path := fmt.Sprintf("registries/%s/types", *templateRegistry)
+		if *regexString != "" {
```

Comment:
I'm fine with this change, but I thought we were working on the new client and all these client side changes are throwaway code?